Shopist/Shopist/Views/CollectionViewCells.swift

```diff
@@ -0,0 +1,49 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import UIKit
+import AlamofireImage
+
+final class Cell: UICollectionViewCell {
+    var text: String? {
+        get { label.text }
+        set { label.text = newValue }
+    }
+    var image: UIImage? {
+        get { imageView.image }
+        set { imageView.image = newValue }
+    }
+    func setImage(url: URL) {
+        imageView.af.setImage(withURL: url)
+    }
+
+    private let imageView: UIImageView = {
+        let imageView = UIImageView(frame: CGRect.zero)
+        imageView.contentMode = .scaleAspectFit
```

Comment:
How about using `.scaleAspectFill` and clipping it to the cell corners? Now it looks buggy:

<img width="333" alt="Screenshot 2020-08-03 at 11 52 43" src="https://user-images.githubusercontent.com/2358722/89170401-d9cee800-d57f-11ea-9034-0aa67dd5fe14.png">

As we don't have design for this, we can use Shopist website and Shopist Android as the reference for UI.

Reply:
i was told nobody would see that UI, have we changed the plan? `contentMode` doesn't change generated rum events

Reply:
IMO it's fair to expect our users to launch the app and browse the instrumentation & code. And it's almost zero effort to make it display well - nothing fancy.